test/acceptance_tests/acceptancetest_high_cpu.md

```diff
@@ -0,0 +1,47 @@
+<!--
+Copyright © 2020 Pilz GmbH & Co. KG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why the apache license?

Reply:
it is same in the other acceptance test. But i changed both to LGPL